FlowCrypt/src/main/java/com/flowcrypt/email/security/SecurityUtils.kt

```diff
@@ -114,14 +114,19 @@ class SecurityUtils {
      * @throws NoKeyAvailableException
      */
     @JvmStatic
-    fun getRecipientsPubKeys(context: Context, emails: MutableList<String>): MutableList<String> {
+    fun getRecipientsPubKeys(context: Context, emails: MutableList<String>): List<String> {
```

Comment:
Since you filter by `!pgpKeyDetails.isExpired && !pgpKeyDetails.isRevoked` then this should be called `getRecipientsActivePubKeys`

Reply:
getRecipientsActivePubKeys --> getRecipientsUsablePubKeys?

Reply:
also good